src/main/java/seedu/address/logic/commands/ResetPaymentsCommand.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ID;
+
+import java.util.Objects;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Id;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Payment;
+
+public class ResetPaymentsCommand extends Command {
```

Comment:
Not exactly required but might it be better for an additional layer of abstraction into a ResetPaymentsCommand extends PaymentCommand extends Command? And instead of markpaid/resetpayments it'll just be something like payment -reset and payment -mark. 

Thoughhh this might add quite a bit of complexity to the parser and tests so it's maybe a good to have in the future

Reply:
i was thinking of adding that abstraction but then I felt that it might get confusing for the users as they have to type more stuff now..